typescript/infra/src/agents/key-utils.ts

```diff
@@ -438,26 +443,25 @@ export async function persistValidatorAddressesToLocalArtifacts(
       validators: fetchedValidatorAddresses[chain].validators, // fresh from aws
     };
   }
-  // Resolve the relative path
-  const filePath = path.resolve(__dirname, '../../config/aw-multisig.json');
   // Write the updated object back to the file
-  fs.writeFileSync(filePath, JSON.stringify(awMultisigAddresses, null, 2));
+  const dir = path.join(__dirname, '../../config');
+  writeJSON(
+    dir,
+    'aw-multisig.json',
+    JSON.stringify(awMultisigAddresses, null, 2),
```

Comment:
did you test this?
you don't need to stringify with these functions

Reply:
fixed